2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -18,7 +18,22 @@ https: //hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 //cspell: enable
 
 function createBookList(books) {
-  // TODO your code goes in here, return the ul element
+  const ulTag = document.createElement('ul');
+
+  books.forEach((book) => {
+    const liTag = document.createElement('li');
+    const pTag = document.createElement('p');
+    const imgTag = document.createElement('img');
+    pTag.textContent = book.title + ' - ' + book.author;
+    imgTag.setAttribute('src', './assets/' + book.title.toLowerCase().replaceAll(' ', '_') + '.jpg');
```

Comment:
That is very smart however, if someone changed the book title the image will break.

Instead we can add the image src to myBooks array (I will add example) then use that src here:
```suggestion
  //assgin image src to image element
    imgTag.src = book.imgSrc;
```

Reply:
your myBooks array will look like:

```  const myBooks = [
    {
      title: 'The Design of Everyday Things',
      author: 'Don Norman',
      isbn: '978-0465050659',
      alreadyRead: false,
      imgSrc: './assets/the_design_of_everyday_things.jpg'
    },
    {
      title: 'The Most Human Human',
      author: 'Brian Christian',
      isbn: '978-1617933431',
      alreadyRead: true,
      imgSrc: './assets/the_most_human_human.jpg'
    },
    {
      title: 'The Pragmatic Programmer',
      author: 'Andrew Hunt',
      isbn: '978-0201616224',
      alreadyRead: true,
      imgSrc: './assets/the_pragmatic_programmer.jpg'
    },
  ];

Reply:
I think,  we are not allowed to change or manipulate the data or array, so i try to solve from this way. But of course, your advice sounds logical. I will care next time 👍